jsonifier/models/models.py

```diff
@@ -1,13 +1,13 @@
-# Copyright 2017 Akretion (http://www.akretion.com)
+# Copyright 2016 Akretion (http://www.akretion.com)
 # Sébastien BEAU <sebastien.beau@akretion.com>
 # Raphaël Reverdy <raphael.reverdy@akretion.com>
-# Copyright 2020 Camptocamp SA (http://www.camptocamp.com)
+# Copyright 2019 Camptocamp SA (http://www.camptocamp.com)
 # Simone Orsi <simahawk@gmail.com>
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 
 import logging
```

Comment:
You've opened this from `gfcapalbo`, could you open it from `therp` or `sunflower` instead?